server/src/test/java/org/elasticsearch/threadpool/ThreadPoolTests.java

```diff
@@ -51,6 +51,13 @@ public void testBoundedByBetweenMinAndMax() {
         assertThat(ThreadPool.boundedBy(value, min, max), equalTo(value));
     }
 
+    public void testOneEighthAllocatedProcessorsMinOne() {
```

Comment:
Let us also rename the test method now that the utility method has been renamed.